gammapy/cube/make.py

```diff
@@ -395,21 +405,59 @@ def make_mask_energy_edisp_bias(self, dataset):
 
         Parameters
         ----------
-        dataset : `SpectrumDataset` or `SpectrumDatasetOnOff`
+        dataset : `MapDataset`, `MapDatasetOnOff`, `SpectrumDataset` or `SpectrumDatasetOnOff`
             Dataset to compute mask for.
 
         Returns
         -------
         mask_safe : `~numpy.ndarray`
             Safe data range mask.
         """
+        edisp = dataset.edisp
+
         if isinstance(dataset, (MapDataset, MapDatasetOnOff)):
-            raise NotImplementedError(
-                "'edisp-bias' method currently only supported for spectral datasets"
-            )
+            if self.position is None:
+                raise AttributeError(
+                    "To compute the energy bias, a position has to be set"
+                )
+            e_reco = dataset.counts.geom.get_axis_by_name("energy").edges
+            edisp = edisp.get_energy_dispersion(self.position, e_reco)
+            counts = dataset.counts.geom
+        else:
+            counts = dataset.counts
 
-        e_min = dataset.edisp.get_bias_energy(self.bias_percent / 100)
-        return dataset.counts.energy_mask(emin=e_min)
+        e_min = edisp.get_bias_energy(self.bias_percent / 100)
+        return counts.energy_mask(emin=e_min)
+
+    def make_mask_bkg_peak_energy(self, dataset):
```

Comment:
Minor: please rename to `make_mask_energy_bkg_peak()`, so that the all energy mask methods start with `make_mask_energy_xxx`